arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -75,7 +75,7 @@ VstCommTask<T>::~VstCommTask() {
 
 template <SocketType T>
 void VstCommTask<T>::start() {
-  LOG_TOPIC("7215f", TRACE, Logger::REQUESTS)
+  LOG_TOPIC("7215f", INFO, Logger::REQUESTS)
```

Comment:
this should remain at level TRACE
```suggestion
  LOG_TOPIC("7215f", TRACE, Logger::REQUESTS)
```